doc/api/http.md

```diff
@@ -750,12 +750,15 @@ See [`writable.cork()`][].
 <!-- YAML
 added: v0.1.90
 changes:
+  - version: v15.0.0
+    pr-url: https://github.com/nodejs/node/pull/33155
+    description: The `data` parameter can now be a `Uint8Array`.
   - version: v10.0.0
     pr-url: https://github.com/nodejs/node/pull/18780
     description: This method now returns a reference to `ClientRequest`.
 -->
 
-* `data` {string|Buffer}
+* `data` {string|Buffer|Uint8Array}
```

Comment:
I'm trying to remember, are other `TypedArray` types supported also? e.g. `Uint16Array`, `Uint32Array`, etc. If so, this should likely be updated to be `TypedArray` rather than just `Uint8Array`

Reply:
According to docs e.g. fs supports this but not http, see https://github.com/nodejs/node/blob/775bf629ddd3b4f9f33ae887a2bd6653aed989d8/lib/_http_outgoing.js#L836-L845